instrumentation/druid-0.2/javaagent/build.gradle.kts

```diff
@@ -0,0 +1,20 @@
+plugins {
+  id("otel.javaagent-instrumentation")
+}
+
+muzzle {
+  pass {
+    group.set("com.alibaba")
+    module.set("druid")
+    versions.set("[0.2.6,)")
+    skip("1.0.30")
+  }
+}
+
+dependencies {
+  library("com.alibaba:druid:0.2.6")
```

Comment:
For simplicity, and since `1.0.0` was released in 2013 I'd recommend going ahead and baselining against `1.0` and renaming the module to `druid-1.0`

Reply:
@trask done, thank you